libraries/AC_Planck/AC_Planck.h

```diff
@@ -66,8 +66,33 @@ class AC_Planck {
   //Get a position, velocity, yaw command
   bool get_posvel_cmd(Location &loc, Vector3f &vel_cms, float &yaw_cd, bool &is_yaw_rate);
 
+  // handle ack/nack message from ACE
+  void handle_planck_ack(const mavlink_message_t &msg);
+
   uint32_t mux_rates(float rate_up,  float rate_down);
 
+  //Returns true if the last command req was actively NACKd or timed out
+  bool was_last_request_rejected() {
```

Comment:
These methods should really be in the `.cpp`

Reply:
moved